bench/bin/throughput.rs

```diff
@@ -251,7 +293,7 @@ async fn main() -> Result<(), std::io::Error> {
                     for word in words.iter().cycle().take(line_count / 20) {
                         count += 1;
                         if count % 10_000 == 0 {
-                            std::thread::sleep(std::time::Duration::from_millis(50));
+                            std::thread::sleep(std::time::Duration::from_millis(12));
```

Comment:
This is still a terrible way to control the rate, but tokio doesn't have sub millisecond timers to allow us to easily control lines written per second.

Reply:
Just for my understanding, it's dropping the wait time so that both writers are writing files at a faster rate?

Reply:
Yeh. we should really refactor it so that it uses some sort of limiter so that it attempts to write a certain number of lines per millisecond, but it's more logic and the original purpose of this bench was mostly to saturate the agent for debugging purposes